projects/plugins/jetpack/_inc/client/traffic/verification-services.jsx

```diff
@@ -152,6 +159,14 @@ class VerificationServicesComponent extends React.Component {
 										href="https://webmaster.yandex.com/sites/"
 									/>
 								),
+								facebook: (
+									<ExternalLink
+										icon={ true }
+										target="_blank"
+										rel="noopener noreferrer"
+										href="https://developers.facebook.com/"
```

Comment:
Can we link to a more specific page on that site?

Reply:
I tried finding one yesterday but unfortunately, Facebook doesn't seem to allow new accounts to immediately become part of Facebook for Developers.

@jonesch - by any chance, would you have a more specific URL?

Reply:
@Aurorum - When you say `specific URL`, are you referring to the Facebook Verification Page here:
https://user-images.githubusercontent.com/1905918/114889842-7799d880-9dd8-11eb-8d13-3c0c1a60dbd2.png

Reply:
Yep.

Reply:
@Aurorum - I ended up on: https://d.pr/i/K6wxen
`https://business.facebook.com/settings/owned-domains/394857463728192?business_id=3948574637281928`
Hope that helps!

Reply:
@jonesch - thank you, that's super helpful! If you don't mind, could you please confirm if `https://business.facebook.com/settings` or `https://business.facebook.com/settings/owned-domains` are valid links?

Reply:
@Aurorum -
Sure, so `https://business.facebook.com/settings` redirects me to `https://business.facebook.com/settings/people/100034905524494?business_id=1327645670709530`
and `https://business.facebook.com/settings/owned-domains` redirects me to `https://business.facebook.com/settings/owned-domains/944572786278995?business_id=1327645670709530`

But there are some 302s on both.
`https://business.facebook.com/settings/owned-domains/` returned a `302` to `https://business.facebook.com/select/?next=https%3A%2F%2Fbusiness.facebook.com%2Fsettings%2Fowned-domains%2F&ext=1620060763&hash=Aflcmo7ylV5mnrMq2rJDnu9jt357xVwVXzLGN6m8oNmTWg` and then that `302`ed to `https://business.facebook.com/settings/owned-domains/?business_id=1327645670709530&global_scope_id=1327645670709530`

Reply:
Thank you!

@jeherve - I feel like https://business.facebook.com/settings is a sensible link to use, do you agree?

Reply:
That link got me to the right place to pull up a domain, so that works for me.